lib/cocoapods/command/lib/lint.rb

```diff
@@ -22,18 +22,17 @@ def self.options
             ['--fail-fast', 'Lint stops on the first failing platform or subspec'],
             ['--use-libraries', 'Lint uses static libraries to install the spec'],
             ['--use-modular-headers', 'Lint uses modular headers during installation'],
-            ['--sources=https://github.com/artsy/Specs,master', 'The sources from which to pull dependent pods ' \
-             "(defaults to #{TrunkSource::TRUNK_REPO_URL}). " \
-             'Multiple sources must be comma-delimited.'],
-            ['--platforms=ios,macos', 'Lint against specific platforms' \
-              '(defaults to all platforms supported by the podspec).' \
-              'Multiple platforms must be comma-delimited'],
+            ["--sources=https://github.com/artsy/Specs,#{Pod::TrunkSource::TRUNK_REPO_URL}", 'The sources from which ' \
+             "to pull dependent pods (defaults to #{Pod::TrunkSource::TRUNK_REPO_URL}). Multiple sources must be " \
+              'comma-delimited'],
+            ['--platforms=ios,macos', 'Lint against specific platforms (defaults to all platforms supported by the ' \
+              'podspec). Multiple platforms must be comma-delimited'],
             ['--private', 'Lint skips checks that apply only to public specs'],
-            ['--swift-version=VERSION', 'The SWIFT_VERSION that should be used to lint the spec. ' \
-             'This takes precedence over the Swift versions specified by the spec or a `.swift-version` file.'],
-            ['--include-podspecs=**/*.podspec', 'Additional ancillary podspecs which are used for linting via :path.'],
+            ['--swift-version=VERSION', 'The `SWIFT_VERSION` that should be used to lint the spec. ' \
+             'This takes precedence over the Swift versions specified by the spec or a `.swift-version` file'],
+            ['--include-podspecs=**/*.podspec', 'Additional ancillary podspecs which are used for linting via :path'],
             ['--external-podspecs=**/*.podspec', 'Additional ancillary podspecs which are used for linting '\
-              'via :podspec. If there are --include-podspecs, then these are removed from them.'],
```

Comment:
removed all `.` from all commands since some were using them and some were not.

Reply:
actually I am going to opt for adding periods everywhere as it looks very weird without them.

Reply:
bah no, CLAide by default has no periods so we are going to be consistent.